labeled_data/technology/ai/generative_ai/application/index.yml

```diff
@@ -1,6 +1,13 @@
 name: Application
-type: Tech-1
+type: Tech-2
 data:
+  labels:
+    - audio
+    - code
+    - image
+    - text
+    - other
+    - video
   platforms:
```

Comment:
Are all the repos below can not be assigned to any above categories? I think they should be set to above categories since there is an `other` sub field.

I think the best way to do this is that `Application` should only have `labels`, all the orgs or repos should be in sub labels.

Reply:
This is beyond my knowledge 😅  . I just plan to standardize the data format in this PR. 
We may invite @PureNatural to open another issue to resolve all the items stunk in index.yml without any label. He can try to involve his classmates from Hubei University into the task~

Reply:
OK~